src/containers/tutor-profile/profile-info/ProfileInfo.jsx

```diff
@@ -56,21 +56,27 @@ const ProfileInfo = () => {
   const accInfo = accountInfoMock.map((item) => (
     <TitleWithDescription
       description={item.description}
-      descriptionStyles={{ typography: 'overline' }}
       key={item.description}
-      style={{ wrapper: { textAlign: 'center' } }}
+      style={styles.profileTitleComp}
       title={item.title}
-      titleStyles={{ typography: { md: 'h5' } }}
     />
   ))
 
   const buttonGroup = !isMyProfile && (
     <Box sx={styles.buttonGroup}>
-      <Button fullWidth size={isDesktop ? 'extraLarge' : 'medium'} variant='containedLight'>
+      <Button
+        fullWidth
+        size={isDesktop ? 'extraLarge' : 'medium'}
```

Comment:
you can use [SizeEnum](https://github.com/ita-social-projects/SpaceToStudy-Client/blob/fe335b7f49365de7aa0476e2d7e5800cc7983c59/src/types/common/enums/common.enums.ts#L10) in types/common for size